js/twister_directmsg.js

```diff
@@ -5,216 +5,196 @@
 
 function requestDMsnippetList(dmThreadList) {
     var followList = [];
-    for( var i = 0; i < followingUsers.length; i++ ) {
+    for (var i = 0; i < followingUsers.length; i++)
         followList.push({username:followingUsers[i]});
-    }
-    for( var i = 0; i < groupChatAliases.length; i++ ) {
+    for (var i = 0; i < groupChatAliases.length; i++)
         followList.push({username:groupChatAliases[i]});
-    }
 
-    twisterRpc("getdirectmsgs", [defaultScreenName, 1, followList],
-           function(req, ret) {processDMsnippet(ret, dmThreadList);}, dmThreadList,
-           function(req, ret) {console.log("ajax error:" + ret);}, null);
+    twisterRpc('getdirectmsgs', [defaultScreenName, 1, followList],
+        function(req, ret) {processDMsnippet(ret, dmThreadList);}, dmThreadList,
+        function(req, ret) {console.log('ajax error:' + ret);}, null
+    );
 }
 
 function processDMsnippet(dmUsers, dmThreadList) {
     dmThreadList.empty();
 
-    for( var u in dmUsers ) {
-        if( dmUsers.hasOwnProperty(u) ) {
+    for (var u in dmUsers) {
+        if (dmUsers[u]) {
```

Comment:
* it's fastest way to check truthy object property existence as I see. try http://jsperf.com/property-detection/14 or https://jsperf.com/object-detection/11 for example.
* this check will only succeed if the value of dmUsers[u] is truthy, _[meaning it’s an object, a non-empty string, a non-zero number that’s not NaN, true, and not null or undefined](http://www.nczonline.net/blog/2010/07/27/determining-if-an-object-property-exists/)_.
  * currently we only pass `ret` from _twisterRpc()_ as `dmUsers` in _processDMsnippet()_ once in the whole code. so I don't see any prototype of this anyway.
    * and since we're going in `for … in` loop for object without prototype I suppose we don't need checking at all here.